config/pf-elastic-collector.yml

```diff
@@ -0,0 +1,150 @@
+pf-elastic-collector:
+  # Defines the host and port the server is listening on.
+  host: "0.0.0.0:8260"
+
+  # Verbose log output option.
+  verbose: true
+
+  # Configure metrics exposition. Both expvar and Prometheus formats are supported. Both can be
+  # configured at the same time. By default, no metrics are exposed.
+  # 'prometheus_host' can only be configured with a 'host:port' pair.
+  # 'expvar_host' can be configured either with a 'host:port' pair or with a Unix Domain Socket path (with a  'unix://' prefix).
+  # When host:port is used, an HTTP server is exposed. The server does not support TLS.
+  # An empty value disables metrics exposition for the corresponding format.
+  #metrics:
+  #  prometheus_host: 'localhost:9090'
+  #  expvar_host: unix:///tmp/collector-metrics.sock
+
+  # Define the suggested upper limit of memory that pf-elastic-collector should apply. Using a lower
+  # amount of memory might trigger garbage collection more often.
+  #memory_limit: 500M
+
+  # Agent authorization configuration. If no methods are defined, all requests will be allowed.
+  auth:
+    # Define a shared secret token for authorizing agents.
+    secret_token: "caue"
```

Comment:
At some point I want to go through and set up proper docker compose secrets for things like this, but I'm ok with leaving this for now. The kibana encryption keys, for example, are hard-coded in this repo.